metrics/src/main/scala/metrics.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2017 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package freestyle
+package metrics
+
+import freestyle._
+import freestyle.config.ConfigM
+import cats.effect.IO
+import cats.implicits._
+
+// Modules
+@module trait Metrics {
+  val default: Default
+}
+
+// Algebras
+@free trait Default {
```

Comment:
Maybe `DefaultMetrics`? or `DefaultMemoryMetrics`? In the latter case, we could get rid of the suffix `Memory` on each method. What do you think?

Reply:
Finally I decided `DefaultMemoryMetrics`. Thanks for the observation. Also I removed the suffix `Memory` from all methods.